beetsplug/fetchart.py

```diff
@@ -329,20 +329,36 @@ def get(self, album, plugin, paths):
         if plugin.maxwidth in self.VALID_THUMBNAIL_SIZES:
             size_suffix = "-" + str(plugin.maxwidth)
 
+        url = None
```

Comment:
This assignment is not necessary.

Reply:
Leaving this out causes a "reference before assignment" error in the tests.

I've refactored this locally so it's not required anymore.